server/service/individualServices/RegardApplication.js

```diff
@@ -42,7 +42,7 @@ exports.regardApplication = function (applicationName, releaseNumber, user, xCor
             if (result['status'] != 200) {
                 resolve({
                     "successfully-connected": false,
-                    "reason-of-failure": "OL_UNKNOWN"
+                    "reason-of-failure": "OL_DID_NOT_REACH_ALT"
```

Comment:
if result['status'] != 200 , also check whether it is 408 to ensure that OL could not able to reach ALT.